bus-mapping/src/evm/storage.rs

```diff
@@ -0,0 +1,37 @@
+//! Doc this
+use crate::evm::EvmWord;
+use crate::Error;
+use std::collections::HashMap;
+
+/// Represents a snapshot of the EVM stack state at a certain
+/// execution step height.
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct Storage(pub(crate) HashMap<EvmWord, EvmWord>);
```

Comment:
I'd probably use `hashbrown`. It's more lightwheight and much faster. It also supports no_std and works with the default `Hasher` trait. And even without it. 
Just a suggestion, but no need to be applied now if you prefer no to.

See: https://github.com/rust-lang/hashbrown